src/main/java/FileSaver.java

```diff
@@ -0,0 +1,77 @@
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.FileWriter;
+
+public class FileSaver {
+    public static final String filePath = "duke.txt";
+
+    public static void loadData(ArrayList<Task> list) throws IOException {
+        File dataFile = new File(filePath);
+        if(dataFile.createNewFile()) {
+            Duke.printLine();
+            System.out.println("\tSince the file does not exist, I have created a file for you.");
+            Duke.printLine();
+        }
+        Scanner dataScanner = new Scanner(dataFile);
+        while(dataScanner.hasNext()) {
+            String data = dataScanner.nextLine();
+            String type = data.substring(0,1);
+            boolean isDone = data.charAt(4) == '1';
+            String content = data.substring(8);
```

Comment:
4 and 8 are like magic numbers. Consider declaring them as Constants first to give a clearer idea